core/shared/src/main/scala/zio/FiberRef.scala

```diff
@@ -237,6 +268,8 @@ final class FiberRef[A] private[zio] (
 
 object FiberRef {
 
+  type Full[Value0, Patch0] = FiberRef[Value0] { type Patch = Patch0 }
```

Comment:
I have also been using `Typed` for the version that exposes all type parameters (not that it matters).

Reply:
We could also use `WithPatch`. Like `Schedule.WithState`